routers/students-router.js

```diff
@@ -0,0 +1,93 @@
+const studentsRouter = require("express").Router();
+const knex = require("knex");
+
+const knexConfig = {
+  client: "sqlite3",
```

Comment:
Same as before why have a knex file if you are going to write this info in here?